clash-ghc/src-ghc/Clash/GHC/LoadModules.hs

```diff
@@ -627,8 +627,10 @@ wantedLanguageExtensions df =
              , LangExt.ExplicitForAll
              , LangExt.ExplicitNamespaces
              , LangExt.FlexibleContexts
+             , LangExt.FlexibleInstances
              , LangExt.KindSignatures
              , LangExt.MagicHash
+             , LangExt.MultiParamTypeClasses
```

Comment:
If we really need these, they should also go into `Test.Tasty.Clash.outputTest'`